mitiq/calibration/calibrator.py

```diff
@@ -61,9 +78,19 @@ def smallest_error(self) -> int:
         errors = self.errors()
         strategy_errors = np.sum(errors, axis=1)
         strategy_errors /= len(errors[0])
-        strategy_id = np.argmin(strategy_errors)
+        strategy_id = int(np.argmin(strategy_errors))
         return strategy_id
 
+    def is_empty(self) -> bool:
+        return np.allclose(
+            self.mitigated, np.zeros((self.num_strategies, self.num_problems))
+        )
```

Comment:
This method may change depending on the code suggestions given in the parent PR.
In particular we probably need to check if all elements of the data matrices have been filled with associated results.